Source/DafnyCore/Coco/Parser.frame

```diff
@@ -68,20 +68,6 @@ public class Parser {
     errDist = 0;
   }
 
-  public void SemErr(string msg) {
```

Comment:
Can you please explain this removal?

Reply:
This method is no longer needed becasue all calls use a signature with an errorId. By deleting it, calls are forced to consider what errorId to use.